frontend/src/test/java/starter/project/frontend/CarControllerTest.java

```diff
@@ -1,21 +1,56 @@
 package starter.project.frontend;
 
-import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.test.web.client.TestRestTemplate;
-import org.springframework.boot.web.server.LocalServerPort;
-import org.springframework.http.ResponseEntity;
+import starter.project.domain.Car;
+import starter.project.domain.Doors;
+import starter.project.domain.Engine;
+import starter.project.domain.Wheels;
+import java.util.ArrayList;
+import java.util.List;
+import static org.junit.Assert.assertEquals;
 
-import java.net.URL;
-import static org.junit.jupiter.api.Assertions.*;
-import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
 
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public class CarControllerTest {
```

Comment:
There should be a few more tests:

POST a json of one car, adds one car the list
POST a second car, and that appends to the list and doesn't destroy it

DELETE removes a car from the list for a given id and no more
DELETE using a list of ids removes all cars in that list and no more

Any more along those lines 👍